modifications/2020-03-12-ASCDB-tagline/add_tagline.py

```diff
@@ -0,0 +1,11 @@
+import qcportal as ptl
+
+client = ptl.FractalClient.from_file()
+
+ds = client.get_collection("ReactionDataset", "ASCDB")
+ds.data.__dict__["tagline"] = (
```

Comment:
Looking at this in this form, I think this is long enough to be a description. We should probably try to cap tagline at 100 characters or so.

Tagline: `A statistically significant subset of the MGCDB84, GMTKN55, and Minnesota 2015B databases.`

@PierMorgante

Reply:
Could you please change it to "A statistically significant database generated from MGCDB84, GMTKN55, and Minnesota 2015B."?

Reply:
Changes made.

Reply:
Thank you!